repos/system_upgrade/el7toel8/actors/bindupdate/actor.py

```diff
@@ -0,0 +1,34 @@
+from leapp.actors import Actor
+from leapp.models import BindFacts, InstalledRedHatSignedRPM
+from leapp.tags import PreparationPhaseTag, IPUWorkflowTag
+from leapp.libraries.actor import updates
+
+
+class BindUpdate(Actor):
+    """
+    Actor parsing facts found in configuration and modifing configuration.
+    """
+
+    name = 'bind_update'
+    consumes = (InstalledRedHatSignedRPM, BindFacts)
+    produces = ()
+    tags = (PreparationPhaseTag, IPUWorkflowTag)
+
+    pkg_names = {'bind', 'bind-sdb', 'bind-pkcs11'}
+
+    def has_package(self, t_rpms):
+        """ Replacement for broken leapp.libraries.common.rpms.has_package """
+        for fact in self.consume(t_rpms):
+            for rpm in fact.items:
+                if rpm.name in self.pkg_names:
+                    return True
+        return False
+
+    def process(self):
+        if not self.has_package(InstalledRedHatSignedRPM):
+            self.log.debug('bind is not installed')
+            return
+
+        for bindfacts in self.consume(BindFacts):
+            modified = updates.update_facts(bindfacts)
+            self.log.info('{0} BIND configuration files modified.'.format(modified))
```

Comment:
Please use logger specific string formatting (https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles)
```
>>> logger.error('This is an%s %s %s', 'other,', 'ERROR,', 'message')
2010-10-28 15:19:29,833 foo.bar ERROR This is another, ERROR, message
>>>
```

Reply:
@ZhukovGreen This should be ok. In our guidelines, the format string is suggested. We have plenty of examples in our code where this is used even for logging.

Reply:
fine to me